contracts/IRelayRecipient.sol

```diff
@@ -32,7 +32,20 @@ contract IRelayRecipient {
      *  @param signature - sender's signature over all parameters except approvalData
      *  @param approvalData - extra dapp-specific data (e.g. signature from trusted party)
      */
-    function acceptRelayedCall(address relay, address from, bytes memory encodedFunction, uint gasPrice, uint transactionFee, bytes memory signature, bytes memory approvalData) public view returns (uint);
+     function acceptRelayedCall(
+        address relay,
+        address from,
+        bytes memory encodedFunction,
+        uint256 transactionFee,
+        uint256 gasPrice,
+        uint256 gasLimit,
+        uint256 nonce,
```

Comment:
It does make sense in the context of using a signed `approvalData` to approve a meta-tx (which is one of the main use cases we have), since the signature should be made over the the nonce as well.

I guess the answer is similar to the `maxCharge`: since it's relatively cheap to throw in that parameter, we have a concrete use case where it makes sense to make it available for the user, and it does not involve adding the EncoderV2, it's best to include it. Especially since it's difficult to prevent which other use cases the app devs may come up with!